cypress/.eslintrc.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-env node */
+
+module.exports = {
```

Comment:
Modeled  off of the .eslinrc.js in `/app/javascript`. The code in here can be the latest and greatest JS since it's just for tests, so I put `'es2021'`. Also, there is Cypress specific eslint configuration/

Reply:
With this, we're going to have 3 eslintrc. Will this have a negative impact on DX?

Reply:
It shouldn't affect DX as this is laser focused on the test folder for Cypress tests only. You can inherit from other eslint configurations, but in this case, the inheritance does not make sense in this scenario.